### DISCUSSION TITLE
Debian 12: Unable to locate package far2l

### DISCUSSION THREAD

**Discussion by tms320** [elfmz/far2l#2271] [Q&A] [answered]
I am trying to install far2l on Debian 12 server (no gui desktop):
```bash
apt install far2l
```
but get error:
```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package far2l
```
What am I doing wrong?

**Comment by akruphi** [comment, marked as answer]
Debian has far2l packages only in **sid-unstable** / **13 trixie-testing** / **12 bookworm-backports**: https://packages.debian.org/search?keywords=far2l and in Debian 12 you should install from backports ( https://backports.debian.org/Instructions/ )

Note: now far2l in official repositories Debian/Ubuntu is only TTY|Xi version with extra dependencies of pair X11-libs. It may be not convenient for some servers. For servers without X and only terminal/ssh access the pure far2l-TTY version is more suitable or you can use portable version from https://github.com/spvkgn/far2l-portable/releases
